src/box/index.h

```diff
@@ -696,6 +694,27 @@ struct index_read_view_iterator {
 		    const char **data, uint32_t *size);
 };
 
+/** Size of the index_read_view_iterator struct. */
+#define INDEX_READ_VIEW_ITERATOR_SIZE 48
+
+static_assert(sizeof(struct index_read_view_iterator_base) <=
+	      INDEX_READ_VIEW_ITERATOR_SIZE);
```

Comment:
AFAIK, we follow C/C++11 standard, and `static_assert` with no message is a [C/C++17 extension](https://en.cppreference.com/w/cpp/language/static_assert), so please provide every such assert with message.

Reply:
Done.

<details><summary>diff</summary>

```diff
diff --git a/src/box/index.h b/src/box/index.h
index cc2b0444d184..7819cf7eef31 100644
--- a/src/box/index.h
+++ b/src/box/index.h
@@ -698,7 +698,9 @@ struct index_read_view_iterator_base {
 #define INDEX_READ_VIEW_ITERATOR_SIZE 48
 
 static_assert(sizeof(struct index_read_view_iterator_base) <=
-	      INDEX_READ_VIEW_ITERATOR_SIZE);
+	      INDEX_READ_VIEW_ITERATOR_SIZE,
+	      "sizeof(struct index_read_view_iterator_base) must be less than "
+	      "or equal to INDEX_READ_VIEW_ITERATOR_SIZE");
 
 /**
  * Iterator over an index read view.
diff --git a/src/box/memtx_hash.cc b/src/box/memtx_hash.cc
index 39fa6198e903..04ec67cfdf88 100644
--- a/src/box/memtx_hash.cc
+++ b/src/box/memtx_hash.cc
@@ -522,7 +522,9 @@ struct hash_read_view_iterator {
 };
 
 static_assert(sizeof(struct hash_read_view_iterator) <=
-	      INDEX_READ_VIEW_ITERATOR_SIZE);
+	      INDEX_READ_VIEW_ITERATOR_SIZE,
+	      "sizeof(struct hash_read_view_iterator) must be less than or "
+	      "equal to INDEX_READ_VIEW_ITERATOR_SIZE");
 
 static void
 hash_read_view_free(struct index_read_view *base)
diff --git a/src/box/memtx_tree.cc b/src/box/memtx_tree.cc
index 7a6c4aae65d1..a92103f1a43f 100644
--- a/src/box/memtx_tree.cc
+++ b/src/box/memtx_tree.cc
@@ -1864,9 +1864,13 @@ struct tree_read_view_iterator {
 };
 
 static_assert(sizeof(struct tree_read_view_iterator<false>) <=
-	      INDEX_READ_VIEW_ITERATOR_SIZE);
+	      INDEX_READ_VIEW_ITERATOR_SIZE,
+	      "sizeof(struct tree_read_view_iterator<false>) must be less than "
+	      "or equal to INDEX_READ_VIEW_ITERATOR_SIZE");
 static_assert(sizeof(struct tree_read_view_iterator<true>) <=
-	      INDEX_READ_VIEW_ITERATOR_SIZE);
+	      INDEX_READ_VIEW_ITERATOR_SIZE,
+	      "sizeof(struct tree_read_view_iterator<true>) must be less than "
+	      "or equal to INDEX_READ_VIEW_ITERATOR_SIZE");
 
 template <bool USE_HINT>
 static void
diff --git a/src/box/sequence.c b/src/box/sequence.c
index 98a87fa76c4e..7f479274cdac 100644
--- a/src/box/sequence.c
+++ b/src/box/sequence.c
@@ -317,7 +317,9 @@ struct sequence_data_iterator {
 };
 
 static_assert(sizeof(struct sequence_data_iterator) <=
-	      INDEX_READ_VIEW_ITERATOR_SIZE);
+	      INDEX_READ_VIEW_ITERATOR_SIZE,
+	      "sizeof(struct sequence_data_iterator) must be less than or "
+	      "equal to INDEX_READ_VIEW_ITERATOR_SIZE");
 
 /** Implementation of next_raw index_read_view_iterator callback. */
 static int
```

</details>